homeassistant/components/rako/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Do not manually edit the translation files. Instead generate it:
```shell
python3 -m script.translations develop
``` 
https://developers.home-assistant.io/docs/internationalization/core#test-translations